bin/migrate-oats-data/applications/submissions/parcels/primary_contact/application_primary_contact.py

```diff
@@ -0,0 +1,139 @@
+from common import (
+    setup_and_get_logger,
+    BATCH_UPLOAD_SIZE,
+    OATS_ETL_USER,
+    ALCSOwnerType,
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor
+
+etl_name = "init_application_primary_contacts"
+logger = setup_and_get_logger(etl_name)
+
+
+@inject_conn_pool
+def init_application_primary_contacts(conn=None, batch_size=BATCH_UPLOAD_SIZE):
+    logger.info(f"Start {etl_name}")
+    with conn.cursor(cursor_factory=RealDictCursor) as cursor:
+        with open(
+            "applications/submissions/sql/parcels/primary_contact/application_primary_contact_count.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            count_query = sql_file.read()
+            cursor.execute(count_query)
+            count_total = dict(cursor.fetchone())["count"]
+        logger.info(f"Total Application data to insert: {count_total}")
+
+        failed_inserts = 0
+        successful_updates_count = 0
+        last_file_number = 0
+
+        with open(
+            "applications/submissions/sql/parcels/primary_contact/application_primary_contact.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            application_sql = sql_file.read()
+            while True:
+                cursor.execute(
+                    f"{application_sql} AND alr_application_id > {last_file_number} ORDER BY alr_application_id;"
```

Comment:
the semicolon exists here, placing it in the sql will break it

Reply:
Ooooh, I see. That makes sense.